torchani/data/new.py

```diff
@@ -0,0 +1,444 @@
+import numpy as np
+import torch
+import functools
+from ._pyanitools import anidataloader
+import pkbar
```

Comment:
Could you make it a
```python
try:
    import pkbar
except ImportError:
    pkbar = None
```
The availability of dataset api should not depend on a progress bar, neither `tqdm` nor `pkbar`.

Reply:
40 seconds loading needs it.

Reply:
Adding a progress bar is good. But just don't add new dependency.

Reply:
make import into a `try` block, and safeguard all access to pkbar with a
```python
if pkbar is not None:
   # do simething
```

Reply:
You are making the code very ugly. pkbar is just one py file.

Reply:
No, keeping dependency minimum is a common approach in software engineering.

Reply:
NO, it's not a software, it's only a research project.

Reply:
Done as you requested.